frontend_app_agenda/src/container/Agenda.jsx

```diff
@@ -117,7 +117,7 @@ class Agenda extends React.Component {
   }
 
   loadAgendaList = async workspaceId => {
-    const { state } = this
+    const { state, props } = this
```

Comment:
you added props but I don't see any usage, shouldn't this create a linting error ?

Reply:
`props` is used on lines 132 and 135, this caused a linting error that was not raised because standard was not enabled for app agenda before.

Reply:
ok, you can resolve